pyzebra/comm_export.py

```diff
@@ -0,0 +1,134 @@
+
+def fill_string(string, total_lenght):

```

Comment:
You can use `str.rjust` function instead (https://docs.python.org/3/library/stdtypes.html#str.rjust)

Reply:
In fact, there are similar padding options in `str.format()` function. Basically, all of the following have similar output:
```
fill_string('asdf', 10)
'asdf'.rjust(10)
'{:>10}'.format('asdf')
```
But I would also suggest you to have a look at f-strings, which appeared in python/3.6. They could be even easier to use in this case.

Reply:
I deduplicated the code and used the fstrings to do the padding